recipes/fontconfig/all/conanfile.py

```diff
@@ -48,9 +48,7 @@ def build_requirements(self):
             self.build_requires("msys2/cci.latest")
 
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version])
-        extrated_dir = self.name + "-" + self.version
-        os.rename(extrated_dir, self._source_subfolder)
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
```

Comment:
I like to track minimum conan version required, but some reviewers disagree.

Reply:
I don't disagree, but I'm too lazy to look up the syntax and version :/
In a perfect world there would be a bot watching pull requests, and it would detect such new constructs, and do a PR on the PR's branch to add the minimal conan version. But of course I'm even lazier to develop this bot 🙄.

Reply:
it's `required_conan_version = ">=1.33.0"`

Reply:
It needs to be a hook and we need to keep updated... or else we will never keep it up to date